components/popup/readme.md

```diff
@@ -12,7 +12,7 @@ To trigger the Popup, it should be paired with an anchor (e.g., a button).
 | `arrow`       | `arrow`       | `boolean \| undefined`                                                                                                                                               | `true`    |
 | `corner`      | `corner`      | ` "top" \| "top-start" \| "top-end" \| "right" \| "right-start" \| "right-end" \| "bottom" \| "bottom-start" \| "bottom-end" \| "left" \| "left-start" | "left-end"` | `"left"`  |
 | `dismissible` | `dismissible` | `boolean \| undefined`                                                                                                                                               | `true`    |
-| `distance`    | `distance`    | `number`                                                                                                                                                             | `10`      |
+| `distance`    | `distance`    | `number`                                                                                                                                                             | `0`      |
```

Comment:
should we allow any number? or expose this as API at all?
maybe it's better just to toggle between attached to anchored element (0px) and if arrowed, having the exact arrow offset.

in what use-case do we need to expose this to authors?